src/_guides/language/language-tour.md

```diff
@@ -284,9 +284,9 @@ Instance variables can be `final` but not `const`.
 
 Here's an example of creating and setting a final variable:
 
-<!-- language-tour/final-initialization/bin/main.dart -->
+<?code-excerpt "language-tour/final-initialization/bin/main.dart"?>
 {% prettify dart %}
-final name = 'Bob'; // Or: final String name = 'Bob';
+final String name = 'Bob'; // Or: final String name = 'Bob';
```

Comment:
Why adding types?

Reply:
Type annotations like these are appearing because the code excerpt is now synced with its originating source file. https://github.com/dart-lang/dart-up-and-running-book/pull/53 is the PR in which most of these type annotations were introduced.

Here is a direct link to the change made to the code excerpt you commented on: https://github.com/dart-lang/dart-up-and-running-book/pull/53/files#diff-9c88f184c7d08c6ebcfb3ff1909212db:

```diff
- final name = 'Bob'; // Or: final String name = 'Bob';
+ final String name = 'Bob'; // Or: final String name = 'Bob';
  // name = 'Alice';  // Uncommenting this causes an error
  // END(final_initialization)
  
- final name2 = 'Roberta';
+ final String name2 = 'Roberta';
  ...
  ```

In this, and many other cases, I believe that type annotations were added because the `type_annotate_public_apis` analyzer rule.

If this isn't what we want, then we'll need to drop that analyzer rule and fix the source files. Let me know what you prefer.

Reply:
This is kinda tough.

If we're just showing folks "here's how you set a variable", that code – IMHO – should be in a `main` method or something.

These are more tricky. v2 inference will mean even public fields – like these – won't strictly need types.

@munificent – thoughts here?

Reply:
> ... that code – IMHO – should be in a main method or something.

Agreed. I can review each of the code diffs (that add extra type annotations), and remove the annotation if it isn't meaningful for that particular example.